vpd-tool/src/vpd_tool_main.cpp

```diff
@@ -25,7 +26,17 @@ int main(int argc, char** argv)
         "        From hardware to console: "
         "vpd-tool -r -H -O <DBus Object Path> -R <Record Name> -K <Keyword Name>\n"
         "        From hardware to file: "
-        "vpd-tool -r -H -O <EEPROM Path> -R <Record Name> -K <Keyword Name> --file <File Path>");
+        "vpd-tool -r -H -O <EEPROM Path> -R <Record Name> -K <Keyword Name> --file <File Path>\n"
+        "Write:\n"
+        "    IPZ Format:\n"
+        "        On dbus: "
```

Comment:
will write on D-bus - by default writes on primary hardware and backup sources as well?

Reply:
yes. 
we can leave this info in write option I guess.

Reply:
Updated a Note for this .